### PR TITLE
Add known issue for sublogger memory leak

### PR DESCRIPTION
**Pull request by davidadeleon**
Add known issue around VAULT-21649

### CODE REVIEW

**Review by ltcarbonell — APPROVED** [review]

File: website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Vault is storing references to ephemeral sub-loggers leading to a memory leak
+
+Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
+a memory leak. This impacts many areas of Vault, but primarily Approle Authentication methods, and logins in Enterprise.
```
Comment (hghaf099) [inline website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx:4]:
```suggestion
a memory leak. This impacts many areas of Vault, but primarily Approle authentication methods, and logins in Enterprise.
```

**Review by hghaf099 — APPROVED** [review]
LGTM!

File: website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+### Vault is storing references to ephemeral sub-loggers leading to a memory leak
+
+Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
+a memory leak. This impacts many areas of Vault, but primarily Approle authentication methods, and logins in Enterprise.
```
Comment (jasonodonnell) [inline website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx:4]: I think the approle leak is a much smaller problem then logins. There's probably more besides approle so I don't think it should be singled out here.

```suggestion
a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
```

**Review by jasonodonnell — APPROVED** [review]

File: website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx (hunk 2 above)
Comment (davidadeleon) [reply website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx:4]: Agreed.

File: website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+### Vault is storing references to ephemeral sub-loggers leading to a memory leak
+
+Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
+a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
+There is no workaround.
+
+#### Affected versions
+
+This issue affects Vault CE and Vault Enterprise versions:
```
Comment (briankassouf) [inline website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx:9]:
```suggestion
This issue affects Vault Community and Enterprise versions:
```

**Pull request merged**